src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/repository/ActionPlanRepository.kt

```diff
@@ -0,0 +1,12 @@
+package uk.gov.justice.digital.hmpps.educationandworkplanapi.app.database.jpa.repository
+
+import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.stereotype.Repository
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.app.database.jpa.entity.ActionPlanEntity
+import java.util.UUID
+
+@Repository
+interface ActionPlanRepository : JpaRepository<ActionPlanEntity, UUID> {
```

Comment:
UUID or String?

Reply:
Definitely `UUID` because the ID field (primary key) of the entity itself is a `UUID` :
```
@Table(name = "action_plan")
@Entity
@EntityListeners(AuditingEntityListener::class)
class ActionPlanEntity(
  @Id
  @GeneratedValue
  @UuidGenerator
  @NotNull
  var id: UUID? = null,
```

Reply:
Ah of course, I was being dumb. Ignore me!